script/copy_deduplicate

```diff
@@ -238,7 +238,8 @@ def get_query_job_configs(client, stable_table, date, dry_run, hourly, priority)
     if hourly:
         stable_table = client.get_table(stable_table)
         for hour in range(24):
-            value = (date + timedelta(hours=hour)).isoformat()
+            dt = (datetime(*date.timetuple()[:6]) + timedelta(hours=hour))
+            value = dt.isoformat()
```

Comment:
the `.isoformat()` isn't needed. When I was writing this I got an error that I thought meant I couldn't pass a `datetime` to a `ScalarQueryParameter` of type `TIMESTAMP`, but it was actually that I was passing a `date` that I only thought was a `datetime`. 😖